docs/merged_objects.md

```diff
@@ -224,8 +224,104 @@ Some merged objects may have some additional sample metadata columns specific to
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs were used |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs were used |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative/isotype control ADTs were not used |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs were not used |
+| `discard`  | Indicates whether the cell should be discarded based on ADT QC statistics |
+
+
+Metrics for each of the ADTs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+rowData(altExp(merged_sce, "adt")) # adt metrics
+```
+
+This data frame contains the following columns with statistics for each ADT:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean-SCPCL000000`      | Mean ADT count across all cells/droplets                       |
+| `detected-SCPCL000000`  | Percent of cells in which the ADT was detected (ADT count > 0 ) |
+| `target_type` | Whether each ADT is a target (`target`), negative/isotype control (`neg_control`), or positive control (`pos_control`). If this information was not provided, all ADTs will have been considered targets and will be labeled as `target` |
+
+Finally, additional metadata for ADT processing can be found in the metadata slot of the alternative experiment
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `metadata` slot as a list.
+Similar to the [parent experiment metadata](#singlecellexperiment-experiment-metadata), the `metadata` will contain three items: `library_id`, `sample_id`, and `library_metadata`.
+The `library_metadata` contains the same list of metadata as its corresponding parent experiment library's metadata and includes one additional field `ambient_profile`, which holds a list of the ambient concentrations of each ADT for the given library.
+
+```r
+metadata(altExp(merged_sce, "adt")) # adt metadata
+```
+
+### Additional SingleCellExperiment components for multiplexed libraries
+
+
+Multiplexed libraries will contain several additional per-cell data columns in the main `colData` data frame (accessed with `colData(merged_sce)` [as above](#singlecellexperiment-cell-metrics)), including cellhash HTO QC statistics and results from demultiplexing analysis.
```

Comment:
Again, I would mention that these were all calculated on each individual library before merging.